### DISCUSSION TITLE
Email templates - missing information in directus docs

### DISCUSSION THREAD

**Discussion by tieftontier** [directus/directus#6568] [Archive]
I'm trying to create custom email templates to enable our users to reset their passwords. In the directus docs it is described how to do it: https://docs.directus.io/guides/email-templates/

**BUT: I am missing the information which variable I have to include --> see screenshot.**

![image](https://user-images.githubusercontent.com/65083798/123872237-185a5780-d935-11eb-8e2e-628f9bafef6e.png)

**It would be awesome if you could add a basic example, what a template would look like.**
_Maybe anyone here can share an example too? It would be awesome and greately appreciated._

Thanks a lot for any help here!
Tom

**Comment by rijkvanzanten** [comment]
Ha that's odd, must be VuePress reading the variable there as an actual variable to render 🤔 

Anyways, here's the source of that page:

https://github.com/directus/directus/blob/main/docs/guides/email-templates.md

The variable is `{{ url }}` 👍🏻 

The best examples for templates are the actual templates that Directus uses by default 🙂 
https://github.com/directus/directus/tree/main/api/src/services/mail/templates

**Reply by tieftontier** [reply]
Thank you so much! That helped a lot!

One thing left:
- In the email the project-logo-image in the header is not loading.

I think it happens because the image-link in the email is set to:
`https://ci5.googleusercontent.com/proxy/4b3zLkutjQe0H4p2NGBgeAG1IyLpibAYivphnxZKW2Z8s9jBj-TamyTe8-Q-dumndj2Qf-ZUFmiF3prfrNiH7FOBYoyW2M9jhDbsqfKgWuK-GoSR=s0-d-e1-ft#http://localhost:8055/assets/f34b9816-87d5-4b83-958f-d9493253f139`
instead of just:
`http://localhost:8055/assets/f34b9816-87d5-4b83-958f-d9493253f139`

Keep up your great support guys! You are awesome!
Tom

**Reply by rijkvanzanten** [reply]
> https://ci5.googleusercontent.com/proxy/

This is most likely caused by whatever email server / client you're using, Directus just sets it to the root URL. That being said, is `http://localhost:8055` accessible through your email client? I think the problem is that your PUBLIC_URL environment variable isn't configured to the actual publicly available URL 🙂
